platforms/documentation/docs/src/docs/userguide/dep-man/getting_started_dep_man.adoc

```diff
@@ -0,0 +1,85 @@
+It is also important when you want to reuse existing libraries, or you need to upgrade those libraries while dealing with their versions.
+
+Let's look at a Java project where the code relies on link:https://mvnrepository.com/artifact/com.google.guava/guava[Guava] which is a suite of Google Core libraries for Java.
+The build file of the project includes the following:
+
+====
+include::sample[dir="snippets/dependencyManagement/introductionDependencies/kotlin",files="build.gradle.kts[tags=dependency-intro]"]
+include::sample[dir="snippets/dependencyManagement/introductionDependencies/groovy",files="build.gradle[tags=dependency-intro]"]
+====
+
+Within the `dependencies` block, there are three things to notice when a dependency is declared:
+
+1. The configuration: `implementation` also known as the scope the dependency is applied to
+2. The coordinates: `com.google.guava:guava` made up of a `group` and an artifact `name` which are uniquely identifiable
+3. The version: `32.1.2-jre` which is only required for published dependencies
+
+Dependencies can be local or remote.
+To let Gradle know where to find the dependencies declared, you can use the `repositories{}` block in the <<build_file_basics.adoc#build_file_basics,Build File>>.
+
+Let's expand our example:
+
+====
+include::sample[dir="snippets/dependencyManagement/introductionDependencies/kotlin",files="build.gradle.kts[tags=dependency-repo]"]
+include::sample[dir="snippets/dependencyManagement/introductionDependencies/groovy",files="build.gradle[tags=dependency-repo]"]
+====
+
+In this example, Gradle fetches the `guava` and `juneau-marshall` _dependencies_ from the link:https://repo1.maven.org/maven2/[Maven Central] and link:https://maven.google.com/[Google] _repositories_.
+
+== 1. Declaring Dependencies
+
+You can add code to your Java project from an external library such as `com.google.common.base` (a Guava package) which becomes a _dependency_. +
+
+== 2. Understanding Configurations
+
+A build engineer can <<declaring_dependencies.adoc#declaring-dependencies,declare dependencies>> for different scopes. For example, you can declare dependencies that are only used at compile time.
+Gradle calls the <<declaring_dependencies.adoc#sec:what-are-dependency-configurations,scope of a dependency>> a _configuration_.
+
+== 3. Declaring Repositories
+
+You can <<declaring_repositories.adoc#declaring-repositories,declare _repositories_>> to tell Gradle where to fetch local or remote _dependencies_. +
+
+During a build, Gradle locates and downloads the dependencies, a process called <<dependency_resolution.adoc#sec:how-gradle-downloads-deps,_dependency resolution_>>.
+
+== 4. Centralizing Dependencies
+
+Centralizing dependencies in Gradle involves managing dependencies in a single location to ensure consistency and simplify dependency management across multiple projects.
+Some concepts include:
+
+- A lockfile is a file that stores the exact versions of dependencies used in a project
```

Comment:
Should talk about platforms here

Reply:
This is the old intro. I have updated it.